test/Microsoft.AspNetCore.Mvc.Core.Test/Formatters/TextInputFormatterTest.cs

```diff
@@ -383,8 +385,46 @@ public void GetSupportedContentTypes_NonNullContentType_FiltersContentTypes()
             Assert.Collection(results, c => Assert.Equal("text/xml", c));
         }
 
-        private class TestFormatter : InputFormatter
+        [Fact]
```

Comment:
I'd expect some of these tests to still apply to `InputFormatter`. Split em up as appropriate.